api/k0smotron.io/v1beta1/k0smotroncluster_types.go

```diff
@@ -101,6 +101,8 @@ type ClusterSpec struct {
 	//+kubebuilder:default={"image":"quay.io/k0sproject/etcd:v3.5.13","persistence":{}}
 	Etcd EtcdSpec `json:"etcd,omitempty"`
 
+	// TopologySpreadConstraints will be passed directly to BOTH etcd and .
```

Comment:
and what? looks like some words missing :D

Reply:
And it's classified :)  
Yeah, nice catch, fixed!